include/pagmo/population.hpp

```diff
@@ -411,6 +432,23 @@ class population
     }
 
 private:
+    // Short routine tp update the champion. Does nothing if the problem is MO
+    inline void update_champion(const vector_double& x, const vector_double& f)
```

Comment:
does it not mean .that the function can be just copied where called rather than invoked?